src/V3Tristate.cpp

```diff
@@ -412,6 +412,47 @@ class TristateVisitor final : public TristateBaseVisitor {
         }
         return VN_AS(invarp->user1p(), Var);
     }
+    AstConst* getEnConstp(AstConst* const constp) {
```

Comment:
```suggestion
    AstConst* getNonZConstp(AstConst* const constp) {
```

As elsewhere in this visitor En generally refers to the enable expression being built.